cypress/support/pages/cart.pageObject.js

```diff
@@ -0,0 +1,22 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  url = '/cart.html';
+
+  clickAddToCart() {
```

Comment:
A function's name should say what the function does. Either split it into two functions or name it e.g. `clickAddToCartAndAssertAlert()`.